lib/webdrivers/edge_finder.rb

```diff
@@ -24,6 +24,8 @@ def location
       private
 
       def user_defined_location
+        return unless defined? Selenium::WebDriver::Edge.path
```

Comment:
Why do you need this line? If it isn't defined it probably isn't going to do the right thing.

Reply:
This permits people using this gem and slightly older versions of selenium. This way you don't get crashes, it just fails gracefully and then attempts to find the location using the System platform location methods.

Reply:
The versions of Selenium that don't have this method don't support Chromium Edge, so it shouldn't matter. Do you have a specific scenario where an error was raised, but getting a nil would have done the right thing?

Reply:
Hah, I thought `#path` had been in there longer. It was added for alpha 7. You should be able to add this and have it work in alpha 6:
```
Selenium::WebDriver::Edge = Selenium::WebDriver::EdgeChrome
```

Reply:
Yep, but is there any harm in adding this as a preventative measure, given it doesn't change what the function does.